test/PullRequests.test.js

```diff
@@ -0,0 +1,16 @@
+const PullRequests = require('../src/PullRequests')
+
+test('.isIgnorable(pr) returns true with matched strings', () => {
+  const pullRequest = new PullRequests()
+  const pullRequests = [
+    { title: "Dont merge - this is a PR title" },
+    { title: "Don't merge - this is a PR title" },
+    { title: "Do not merge - this is a PR title" },
+    { title: "[DO NOT MERGE] - this is a PR title" },
+    { title: "WIP - this is a PR title" }
+  ]
+
+  for (let pr of pullRequests) {
```

Comment:
Ah right, thanks for pointing that out! Updated here https://github.com/ohbarye/review-waiting-list-bot/pull/18/commits/9945d20d9f80240808fc1dccd724561aff5cc546